### DISCUSSION TITLE
Spectre console for a c# version of lazygit?

### DISCUSSION THREAD

**Discussion by RollsChris** [spectreconsole/spectre.console#1347] [Q&A] [answered]
Hi, 
if anyone could answer these questions id be grateful 🐱‍🚀

what do you think about using spectre console for a C# version of lazygit?
or How would you go about it if you wanted to use C#? 
Is C# even a good choice for such a thing?

Maybe I'm thinking about it wrong, I'm not sure how they do the windows opening within the terminal etc.

Thanks

**Comment by nils-a** [comment, marked as answer]
What you are looking for, is a tui library. Have a look at https://github.com/gui-cs/Terminal.Gui.

Honestly, I would not re-implement such a brilliant tool simply to have it in another language.

**Reply by RollsChris** [reply]
I'm just looking for projects to keep my C# skills sharp. My job is very limited at the minute ;( 
thanks ill check the tui library  out.

I have also asked how to get started with developing lazygit as an opportunity to learn go, i think it would be a good project to help learn it.

Thanks for the input
